v2_bridges/erc20_on_int/Cargo.toml

```diff
@@ -13,6 +13,12 @@ authors = ["Greg Kapka <gregkapka@gmail.com>"]
 # NOTE: Turns off block header & chain validation.
 non-validating = ["common_eth/non-validating"]
 
+# NOTE: Normally the peg-in event coming from the erc20 chain will have origin details pertaining _to_
+# that erc20 chain. With this feature enabled, we instead look for the original minting action on the
+# erc20 chain (if extant) and gather originating tx details from that instead.
```

Comment:
```suggestion
# NOTE: Normally the peg-out event coming from the interim chain will have origin details pertaining _to_ that interim chain. With this feature enabled, we instead look for the metadata event fired from the interim chain (if extant) and gather originating tx details from that instead.
```

Reply:
Also, this means that the comment on my original PR #18 is also wrong, since it was from when we were intending to use the minting/burning events for this metadata. I'll sort that on mine now.

Reply:
Done here: https://github.com/pnetwork-association/ptokens-core/pull/18/commits/4c29973310eb7e15529206e41fcdd1d0d918b5a4